backend/lucene/src/main/java/org/hibernate/search/backend/lucene/lowlevel/docvalues/impl/AbstractSortedNumericDocValues.java

```diff
@@ -14,6 +14,9 @@
  * {@link DocIdSetIterator} APIs. This impl is safe to use for sorting and
  * aggregations, which only use {@link #advanceExact(int)} and
  * {@link #docValueCount()} and {@link #nextValue()}.
+ * <p>
+ * Copied and adapted from {@code PUT THE ORIGINAL CLASS NAME HERE}
```

Comment:
I have removed "AbstractSortedNumericDocValues.java". I use a different solution. Everything is "DoubleMultiValuesSource" and "LongMultiValuesSource". This can sort and aggregate.